yb-voyager/cmd/analyzeSchema.go

```diff
@@ -593,6 +593,72 @@ func checker(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	}
 }
 
+func checkRedundantIndexes() {
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		utils.ErrExit("error getting migration status record: %v", err)
+	}
+	if !msr.MigrationAssessmentDone { //TODO: add the case to handle if assessment is run in export schema
+		return
+	}
+
+	if msr.SourceDBConf.DBType != POSTGRESQL {
+		return
+	}
+
+	//Only considering the case where assessment is run in this migration
+	//If not via assess-migration, there is going to be a change where internally in the export-schema we will run assessment
+	//so we should have this information most of the time.
+	assessmentReportPath := filepath.Join(exportDir, "assessment", "reports", fmt.Sprintf("%s.json", ASSESSMENT_FILE_NAME))
```

Comment:
Just scanned the code, we seem to have too many instances of this logic is duplicated.
Can we have a function? 
We don't need to update all the existing usages, just going forward, it will help.